src/components/SchoolCard.tsx

```diff
@@ -29,38 +36,52 @@ const Wrapper = styled(Card)`
     margin: 1em 0;
   }
   .bottom {
-    flex-shrink: 0;
+    flex: 1 0 auto;
     text-align: right;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: flex-end;
+    @media (min-width: 1100px) {
+      margin-left: 1em;
+    }
   }
   .content {
     flex: 1 0 auto;
+    @media (min-width: 1100px) {
+      flex: 0 1 720px;
+    }
   }
   a {
     text-decoration: underline;
   }
 `;
 
 type SchoolCardProps = {
-  school: any;
+  school: School;
 };
 
-const SchoolCard = (props: SchoolCardProps) => (
-  <Wrapper>
-    <div className="content">
-      <span className={'school-type'}>
-        Szkoła {!props.school.is_public && 'nie'}publiczna
-      </span>
-      <h4>
-        <Link to={`/school/${props.school.id}`}>
-          {props.school.school_name}
-        </Link>
-      </h4>
-      <span className={'district'}>{props.school.address.district}</span>
-    </div>
-    <div className="bottom">
-      <Link to={`/school/${props.school.id}`}>Więcej</Link>
-    </div>
-  </Wrapper>
-);
+const SchoolCard: FC<SchoolCardProps> = ({ school: { is_public, id, school_name, address } }) => {
+  const { isSchoolFavourite, toggleFavouriteSchool } = useFavouriteSchools();
+  const schoolId = id.toString();
+  return (
+    <Wrapper>
+      <div className="content">
+        <span className="school-type">Szkoła {!is_public && 'nie'}publiczna</span>
+        <h4>
+          <Link to={`/school/${id}`}>{school_name}</Link>
+        </h4>
+        <span className="district">{address.district}</span>
+      </div>
+      <div className="bottom">
```

Comment:
Since this element is now on the left, "bottom" as a class name can be a bit confusing. IDK if the class name should contain position of the element at all? :thinking:

Reply:
changed that to side-content and added annotation for content class as `main-content`